### ISSUE TITLE
Custom Header Overrides app-header-background-color defined in themes

### ISSUE THREAD

**Issue by Knapoc** [maykar/custom-header#106]
When the app-header-background-color is defined in a theme, custom header overrides it with the primary-color. As a result some themes are not working as expected. see: https://github.com/home-assistant/home-assistant-polymer/issues/4348

**Comment by Knapoc** [comment]
I just found out that you can define the variable with the following:
```
custom_header:
  compact_mode: true
  background: var(--app-header-background-color);
```

sorry for any inconveniences. Issue closed

**Comment by maykar** [comment]
No inconvenience at all, in fact I'm going to open this one back up as that variable should take precedence over primary color. I just hadn't added yet it since it's a new theme var.

Thanks!

**Comment by maykar** [comment]
This has been added or fixed in the latest version 1.2.2. After updating be sure to check your browser's dev console (usually F12) to make sure you are running the latest version. If it doesn't display 1.2.2 you may either need to clear your cache or reinstall.
